primitives/Cargo.toml

```diff
@@ -2,12 +2,19 @@
 arbitrary = { default-features = false, optional = true, version = "1.0" }
 frame-support = { branch = "moonbeam-polkadot-v0.9.19", default-features = false, git = "https://github.com/purestake/substrate" }
 frame-system = { branch = "moonbeam-polkadot-v0.9.19", default-features = false, git = "https://github.com/purestake/substrate" }
+orml-currencies = { branch = "2022-06-v0.9.19", default-features = false, git = "https://github.com/zeitgeistpm/open-runtime-module-library" }
 orml-tokens = { branch = "2022-06-v0.9.19", default-features = false, git = "https://github.com/zeitgeistpm/open-runtime-module-library" }
 orml-traits = { branch = "2022-06-v0.9.19", default-features = false, git = "https://github.com/zeitgeistpm/open-runtime-module-library" }
-orml-currencies = { branch = "2022-06-v0.9.19", default-features = false, git = "https://github.com/zeitgeistpm/open-runtime-module-library" }
-parity-scale-codec = { default-features = false, features = ["derive", "max-encoded-len"], version = "3.0.0" }
-scale-info = { version = "2.1.1", default-features = false, features = ["derive"] }
-serde = { default-features = false, features = ["derive"], optional = true, version = "1.0.136" }
+parity-scale-codec = { default-features = false, features = [
+  "derive",
+  "max-encoded-len",
+], version = "3.0.0" }
```

Comment:
I tried this, but then it does not break longer array like we defined features

Reply:
I tried but I could not get taplo generate format as you proposed. setting line length very high is no good.
if taplo is not satisfying our needs then we may try something even simpler https://github.com/segeljakt/toml-fmt
toml-fmt formats things more or less according to cargo style guide mentioned above.
however it may require some more edits in script to have --check like functionality as above tool works on STDIN only.

Reply:
That's too bad. I guess the best we can do is this:

```toml
array_auto_collapse = false
array_auto_expand = false
align_comments = false
reorder_keys = true
```

This doesn't do any line breaks, but it still reorders keys and fixes whitespace issues. We will have to enforce correct line breaks manually.

I raised an issue in taplo: https://github.com/tamasfe/taplo/issues/304

Reply:
@maltekliemann should we go ahead with above configs? Do you want max line length specified?

Reply:
Yeah, I'm fine with the config I posted above. It's not ideal, but there might be a solution in the future. I don't think the max line length still matters, so I'm okay to just leave at default.